### PR TITLE
MaynardL_changes in examples and a few issues

### PR DESCRIPTION
**Pull request by LaurieMaynard**
I changed the first two examples and challenges so that it uses a real dataset of bioluminescence extracted from Zuur's book : Mixed Effects Models and Extensions in Ecology with R. The Dataset was added into the data folder for each presentation, so it will likely need to be added to the HTML.

I also solved a few specific issues.

### CODE REVIEW

File: pres-fr/workshop08-pres-fr.Rmd [hunk 1]
```diff
@@ -48,7 +48,7 @@ if(length(new.packages) > 0) {
   }
 ```
 
-# � propos de cet atelier
+# ? propos de cet atelier
```
Comment (katherinehebert) [inline pres-fr/workshop08-pres-fr.Rmd]: Small typo here, ? should be À

File: pres-fr/workshop08-pres-fr.Rmd [hunk 2]
```diff
@@ -154,8 +154,50 @@ En revanche, les **GAM** font cela en ajustant une **fonction de lissage non-lin
 class: inverse, center, middle
 
 ## 2. Introduction aux GAMs
+---
+# Generalized Additive Models (GAM)
+
+Examinons un exemple! Nous allons utilsé le jeu de données `ISIT`.
```
Comment (katherinehebert) [inline pres-fr/workshop08-pres-fr.Rmd:160]: something happened with the é here

File: pres-fr/workshop08-pres-fr.Rmd [hunk 3]
```diff
@@ -154,8 +154,50 @@ En revanche, les **GAM** font cela en ajustant une **fonction de lissage non-lin
...
+
+```{r, eval = FALSE, echo = TRUE}
+isit <- read.csv("data/ISIT.csv",sep=';')
+head(isit)
+```
+
+Ce jeu de donnée comporte des mesures de bioluminescence en relation � la profondeur (*depth*), la station de rechercher et la saison (*Season*).
```
Comment (katherinehebert) [inline pres-fr/workshop08-pres-fr.Rmd]: é character problem here too (actually it occurs a few times through the rest of the document, but I don't want to spam comments. Could you please check this?

**Review by katherinehebert — COMMENTED** [review]
So happy to see the switch over to AIC and to a dataset that is less abstract! This will definitely help to improve the clarity of the material

File: pres-fr/workshop08-pres-fr.Rmd (hunk 3 above)
Comment (LaurieMaynard) [reply pres-fr/workshop08-pres-fr.Rmd]: Yes I will, this is a problem with my french computer, the accent doesn't show in R (they show in this weird characters. I will try to fix it.

**Comment by LaurieMaynard** [comment]
So, when I opened the script, only the _? propos de cet atelier_ was wrong, and all the other characters showed an accent. I have had this problem several times were when I put an accent in a R script, the next time I open it, these weird chracters appear. I managed it pretty well last time I develop a workshop so I am not sure what happened here. So I pushed it again, let me know if you still see the weird characters.

**Comment by katherinehebert** [comment]
> So, when I opened the script, only the _? propos de cet atelier_ was wrong, and all the other characters showed an accent. I have had this problem several times were when I put an accent in a R script, the next time I open it, these weird chracters appear. I managed it pretty well last time I develop a workshop so I am not sure what happened here. So I pushed it again, let me know if you still see the weird characters.

Yay, this is now fixed! Thanks! Good to go. 🎉

**Review by katherinehebert — APPROVED** [review]

**Pull request merged**